### PR TITLE
Feature/fix unknown arch triplet

### LINKED ISSUE #2840: 1.3 regression: AutoToolsBuildEnvironment now aborts on self-defined architectures
With 1.3 our AutoTools packages started to fail on architectures that we define ourselves in settings.yml, these includes: `aarch64`, `armv7hf-sysv` etc. The error message we get is:

`conans.errors.ConanException: Unknown 'armv7hf-sysv' machine. Conan doesn't know how to translate it to the GNU triplet, please report at https://github.com/conan-io/conan/issues`

This is a breaking change and while the warning is probably good it breaks our current packages without changes to conan. Ideally this warning should be not an error but a warning to not break current packages and maybe make it configurable in the future.

### PR DESCRIPTION
**Pull request by memsharded**
- [x] Refer to the issue that supports this Pull Request.
Fix #2840

### CODE REVIEW

**Comment by memsharded** [comment]
It has been released as an independent tool this 1.3 release, hasn't it? 
And if it raises when used as a tool, it is equally inconvenient for users that will have their custom architectures.

File: conans/client/tools/oss.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def get_cross_building_settings(settings, self_os=None, self_arch=None):
     return build_os, build_arch, host_os, host_arch
 
 
-def get_gnu_triplet(os, arch, compiler=None):
+def get_gnu_triplet(os, arch, compiler, output):
```
Comment (lasote) [inline conans/client/tools/oss.py]: Compiler was optional!!

Comment (lasote) [reply conans/client/tools/oss.py]: Is output used?

File: conans/test/util/tools_test.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ def get_values(this_os, this_arch, setting_os, setting_arch, compiler=None):
 
         # Compiler not specified for os="Windows"
         with self.assertRaises(ConanException):
-            tools.get_gnu_triplet("Windows", "x86")
+            tools.get_gnu_triplet("Windows", "x86", None)
```
Comment (lasote) [inline conans/test/util/tools_test.py]: Remove the none

**Pull request merged**